lib/navigation/preprocess_encoders.cpp

```diff
@@ -19,6 +34,7 @@ core::EncoderData EncodersPreprocessor::processData(const core::EncoderData)
   return {0, 0, 0, 0};
 }
 
+/*
 core::EncoderData EncodersPreprocessor::detectOutliers(const core::EncoderData encoder_data)
 {
   /*
```

Comment:
this is why build is failing. you've opened up a new comment block within a comment block and the compiler is being incredibly pedantic